evidential-gene/1.0.0/Dockerfile

```diff
@@ -0,0 +1,37 @@
+################## BASE IMAGE ######################
+FROM biocontainers/biocontainers:v1.0.0_cv4
+
+################## METADATA ######################
+LABEL base_image="biocontainers:v1.0.0_cv4"
+LABEL version="1"
+LABEL software="EvidentialGene"
+LABEL software.version="22jan14"
```

Comment:
Software label should match with directory name, same for version. 

So dickerfile should be in EvidentialGene/22jan14